modules/authentication/src/config/config.ts

```diff
@@ -3,6 +3,15 @@ export default {
     format: 'Boolean',
     default: true,
   },
+  clients: {
```

Comment:
missing anonymous clients setting and behaviour.
"off" is not an argument.
missing multiple clients login setting